### PR TITLE
Image gallery feature

### PR DESCRIPTION
**Pull request by machinesandpixels**

### CODE REVIEW

File: src/components/FavoritesColumn.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styles from '../../styles/ImageGallery.module.css';
+
+const FavoritesColumn = () => (
+
```
Comment (scullenDev) [inline src/components/FavoritesColumn.tsx:5]: Before you create your release branch, be sure to clean up little stylistic inconsistencies, like extra line breaks or missing line breaks.

File: src/components/ImageGallery.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Card from './Card';
+import styles from '../../styles/ImageGallery.module.css';
+
+const ImageGallery = () => (
+  <section className={styles.gallery}>
+    <Card />
```
Comment (scullenDev) [inline src/components/ImageGallery.tsx:7]: Will there always be 9 of these? If so, preferably generate with a map based on an array (maybe just `[1, 2, 3, etc])`, to DRY things up the syntax front. You could then use the array member as the component `key`.

**Review by scullenDev — APPROVED** [review]
Some very small suggestions, but these items (if retained) can be fixed in a later branch. Nice work!

File: src/components/ImageGallery.tsx (hunk 2 above)
Comment (machinesandpixels) [reply src/components/ImageGallery.tsx:7]: Oh, no! There will only be one card eventually that then gets mapped through all the data from Redux. 
These are just placeholders for now. I needed to test that the scroll was visible and so I just copied and pasted it 9 times.

File: src/components/FavoritesColumn.tsx (hunk 1 above)
Comment (machinesandpixels) [reply src/components/FavoritesColumn.tsx:5]: Ok, I will look into it as I missed those inconsistencies. I didn't see  any linting errors but I  will make sure to look at those extra and missing line breaks.

**Pull request merged**